Dockerfile

```diff
@@ -37,11 +37,11 @@ ARG VCS_REF
 ARG VERSION="5.0.0-SNAPSHOT"
 
 LABEL org.label-schema.build-date=${BUILD_DATE} \
-      org.label-schema.name="exist-docker" \
+      org.label-schema.name="existdb" \
       org.label-schema.description="minimal exist-db docker image with FO support" \
```

Comment:
Can we capitalize "Minimal" please?

Reply:
Actually -> "Minimal eXist-db Docker image with FO support"

Reply:
done